Assets/BossRoom/Scripts/Server/Game/Entity/ServerRemoveChildrenOnFaint.cs

```diff
@@ -0,0 +1,48 @@
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    public class ServerRemoveChildrenOnFaint : NetworkBehaviour
```

Comment:
is this the workaround we were talking about? might be worth talking about the workaround in comments

Reply:
No, this is purely gameplay-related behaviour. Once a player dies, they should drop their parented NetworkObject.